exonum/tests/websocket/main.rs

```diff
@@ -99,6 +100,24 @@ fn test_send_transaction() {
         })
     );
 
+    // Send invalid transaction.
+    let (pk, sk) = gen_keypair();
+    let tx = CreateWallet::new(pk, "Bob").sign(SERVICE_ID + 1, pk, &sk);
+    let tx_json =
+        serde_json::to_string(&json!({ "type": "transaction", "payload": { "tx_body": tx }}))
+            .unwrap();
+    client.send_message(&OwnedMessage::Text(tx_json)).unwrap();
+
+    // Check response on sent message.
+    let resp_text = recv_text_msg(&mut client).unwrap();
+    assert_eq!(
+        serde_json::from_str::<serde_json::Value>(&resp_text).unwrap(),
```

Comment:
Nit: Could make sense to introduce a `let response: Value = ...` binding for this expression for the code to be more readable.